BragiCode/src/main/java/frc/robot/subsystems/ShooterSubsystem.java

```diff
@@ -0,0 +1,78 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+package frc.robot.subsystems;
+
+import com.ctre.phoenix6.controls.StrictFollower;
+import com.ctre.phoenix6.hardware.TalonFX;
+
+
+import edu.wpi.first.wpilibj.Timer;
+import edu.wpi.first.wpilibj.smartdashboard.SmartDashboard;
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+import frc.robot.Constants;
+
+
+public class ShooterSubsystem extends SubsystemBase {
+  public TalonFX mShooterMotorLeft;
+  public TalonFX mShooterMotorRight;
+
+  public ShooterState mShooterState;
+
+  private Timer mTimer = new Timer();
+
+  public enum ShooterState {
+  S_Shoot, S_WaitingForFeeder
+}
+
+  /** Creates a new ShooterSubsystem. */
+  public ShooterSubsystem() {
+    mShooterMotorLeft = new TalonFX(Constants.ShooterConstants.kShooterMotorLeft);
+    mShooterMotorRight = new TalonFX(Constants.ShooterConstants.kShooterMotorRight);
+
+    mShooterMotorRight.setControl(new StrictFollower(mShooterMotorLeft.getDeviceID()));

```

Comment:
Do we want a StrictFollower here? 

Usually with motors that are left and right, the motors are spinning in opposite directions. That would mean we need a regular Follower with `OpposeMasterDirection:true`.

Reply:
Fixed